test/specs/products.js

```diff
@@ -162,40 +191,114 @@ describe('Products Page Tests', () => {
-                // console.log("type of: ", typeof Number(productsPrices[i].replace('$','')))
+            for (let i = 0; i < dictValues.length; i++) {
+                ascendingExpectedProductsPrices.push(Number(dictValues[i].replace("$", "")));
             }
-            sortedProductsPrices.sort((a, b) => a - b);
-            console.log("sorted prices: ", sortedProductsPrices);
+            ascendingExpectedProductsPrices.sort((a, b) => a - b);
 
-            // let dictValues = Object.values(ProductPrices)
-            // // expectedProductPrices.push(price)
-            // console.log('expected prices: ', dictValues)
+            expect(ascendingProductsPrices).toEqual(ascendingExpectedProductsPrices);
+            await ProductsPage.filterData('nameAscending')
+        });
+    })
+
+    describe('Footer Tests', () => {
 
-            // let dictValues = Object.values(ProductPrices)
-            // // expectedProductPrices.push(price)
-            // console.log('expected prices: ', dictValues)
+        beforeEach(async function () {
+            await LoginPage.open();
+            await LoginPage.login(CREDENTIALS.standard, CREDENTIALS.password);
+        });
+
+        afterEach(async function () {
+            await ProductsPage.logout();
+        });
 
-            // let dictValues = Object.values(ProductPrices)
-            // // expectedProductPrices.push(price)
-            // console.log('expected prices: ', dictValues)
+        it("check Footer UI elements", async () => {
 
-            // let dictValues = Object.values(ProductPrices)
-            // // expectedProductPrices.push(price)
-            // console.log('expected prices: ', dictValues)
+            expect(await ProductsPage.twitterIcon.isDisplayed()).toBe(true);
+            expect(await ProductsPage.facebookIcon.isDisplayed()).toBe(true);
+            expect(await ProductsPage.linkedInIcon.isDisplayed()).toBe(true);
+            expect(await ProductsPage.copyright.isDisplayed()).toBe(true);
+            expect(await ProductsPage.copyright.getText()).toContain("© 2022 Sauce Labs. All Rights Reserved. Terms of Service | Privacy Policy");
+            expect(await ProductsPage.robotFooter.isDisplayed()).toBe(true);
+            expect(await ProductsPage.robotFooter.getAttribute('src')).toEqual(ImageSource.footerRobot);
         });
+
+        it("the 'Twitter' icon should redirect to the 'https://twitter.com/saucelabs' url", async () => {
+            await ProductsPage.twitterIcon.waitForClickable()
+            await ProductsPage.twitterIcon.click()
+            await browser.switchWindow("https://twitter.com/saucelabs")
+
+            expect(await browser.getUrl()).toEqual("https://twitter.com/saucelabs");
+            await ProductsPage.closeWindowAndSwitchBack()
+        })
+
+        it("the 'Facebook' icon should redirect to the 'https://www.facebook.com/saucelabs' url",
+            async () => {
```

Comment:
keep the it line on the same line. shorten the it description if.

Reply:
modification made